README.md

```diff
@@ -328,33 +328,32 @@ This library provides functions for working with Data Unions.
 
 TODO: check all this documentation before merging/publishing, probably some of it is out of date (DU1 era)
 
-Data union functions take a third parameter, `options`, which are either overrides to options given in the constructor, or optional arguments to the function.
-
+TODO: do we need these parameters (and `confirmations` parameter) somewhere? Either add the support, or remove from `DataUnionDeployOptions` interface (currently commented out)
```

Comment:
yea could be a good thing I guess, goes together with gasPrice really. Leaving it out for now is no issue, can make a ticket to backlog to add it back later